tests/posterior_nn_test.py

```diff
@@ -195,9 +198,10 @@ def test_batched_mcmc_sample_log_prob_with_different_x(
     x_o = ones(num_dim) if x_o_batch_dim == 0 else ones(x_o_batch_dim, num_dim)
 
     posterior = inference.build_posterior(
-        sample_with="mcmc",
-        mcmc_method="slice_np_vectorized",
-        mcmc_parameters=mcmc_params_fast,
+        posterior_parameters=MCMCPosteriorParameters(
+            method="slice_np_vectorized",
```

Comment:
actually, the `"slice_np_vectorized"` has been made the default some time ago, thus we actually do not need to set it here anymore. Or we include it in the fixture as commented above.